README.md

```diff
@@ -16,13 +16,24 @@ Please install the package with Julia's package manager:
 ] add CDSAPI
 ```
 
-## Usage
+## Basic usage
+`CDSAPI.jl` will attempt to use CDS credentials using three different methods with the following priority:
 
-Make sure your `~/.cdsapirc` file exists. Instructions on how to create the file for your user account can be found
+    1. direct credentials provided via a specific file
+    2. environmental variables `CDSAPI_URL` and `CDSAPI_KEY`
+    3. default credential file in home directory `~/.cdsapirc`
+
+A valid credential file is a text file with two lines:
+```
+url: https://yourendpoint
+key: your-personal-api-token
+```
```

Comment:
I believe we don't need to promote the `credentialsfromfile` approach in the README, it is an internal detail. As suggested in previous comments, we have 3 user-facing methods:

1. scoped values
2. env variables
3. default dotrc file

Anything in the README that deviates from these methods is a source of confusion.

Reply:
The specification of the format of the credentials file, was not much to promote the `credentialsfromfile` internal api, but to give an example of what the default credentials file should look like instead. I think it is valuable to give users a reference point of what it should look like to help them better navigare the setup process if they are new to the CDSAPI.